apps/tx-signer/env/.env.unit.test

```
@@ -0,0 +1,10 @@
+NETWORK=sandbox
+OLD_MASTER_WALLET_MNEMONIC="motion isolate mother convince snack twenty tumble boost elbow bundle modify balcony"
+FUNDING_WALLET_MNEMONIC="motion isolate mother convince snack twenty tumble boost elbow bundle modify balcony"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FUNDING_WALLET_MNEMONIC key should go before the NETWORK key

(UnorderedKey)